### PR TITLE
Add missing icc profile when saving webp

### PR DESCRIPTION
**Pull request by alon-ne**
When saving webp, set the profile to the one determined by OptimizeICCProfile() (if it was called).

### CODE REVIEW

File: vips/foreign.c [hunk 1]
```diff
@@ -56,11 +57,22 @@ int save_png_buffer(VipsImage *in, void **buf, size_t *len, int strip, int compr
 // https://github.com/libvips/libvips/blob/master/libvips/foreign/webpsave.c#L524
 // https://libvips.github.io/libvips/API/current/VipsForeignSave.html#vips-webpsave-buffer
 int save_webp_buffer(VipsImage *in, void **buf, size_t *len, int strip, int quality, int lossless, int effort) {
+	const char* profile = "none";
```
Comment (Elad-Laufer) [inline vips/foreign.c:60]: I think that setting the profile should happen higher up, we lose control when it's down here.

**Review by Elad-Laufer — CHANGES_REQUESTED** [review]
See inline comment. Also, I suspect that this applies to all "savers" not just webp

File: vips/foreign.c [hunk 2]
```diff
@@ -56,13 +57,23 @@ int save_png_buffer(VipsImage *in, void **buf, size_t *len, int strip, int compr
...
-	return vips_webpsave_buffer(in, buf, len,
+	char* target_profile = NULL;
+	vips_image_get_as_string(in, "optimized-icc-profile", &target_profile);
+
+	int result = vips_webpsave_buffer(in, buf, len,
 		"strip", INT_TO_GBOOLEAN(strip),
 		"Q", quality,
 		"lossless", INT_TO_GBOOLEAN(lossless),
 		"reduction_effort", effort,
+		"profile", target_profile ? target_profile : "none",
```
Comment (Elad-Laufer) [inline vips/foreign.c:68]: "none"?

**Review by Elad-Laufer — COMMENTED** [review]
"none"?

File: vips/foreign.c (hunk 2 above)
Comment (alon-ne) [reply vips/foreign.c:68]: yep

Comment (alon-ne) [reply vips/foreign.c:68]: https://libvips.github.io/libvips/API/current/libvips-colour.html#vips-profile-load

**Review by Elad-Laufer — APPROVED** [review]

**Pull request closed without merging**